### PR TITLE
S103.33 Adds pagespeed to AuthManager

### PR DESCRIPTION
**Pull request by pvaass**

### CODE REVIEW

**Comment by expand-dev** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [GoogleDeveloperAuthManager.php](https://github.com/ExpandOnline/AuthManager/blob/b190584ccd8b5a451fcec49043f6b8476e151d1b/Lib/Google/GoogleDeveloperAuthManager.php): Refactor this file to either declare symbols or cause side effects, but not both. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2036)
1. ![MAJOR][MAJOR] [PagespeedAuthContainer.php](https://github.com/ExpandOnline/AuthManager/blob/b190584ccd8b5a451fcec49043f6b8476e151d1b/Lib/Pagespeed/PagespeedAuthContainer.php): Refactor this file to either declare symbols or cause side effects, but not both. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2036)
1. ![MAJOR][MAJOR] [PagespeedAuthManager.php](https://github.com/ExpandOnline/AuthManager/blob/b190584ccd8b5a451fcec49043f6b8476e151d1b/Lib/Pagespeed/PagespeedAuthManager.php): Refactor this file to either declare symbols or cause side effects, but not both. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2036)
1. ![MAJOR][MAJOR] [MediaPlatform.php](https://github.com/ExpandOnline/AuthManager/blob/b190584ccd8b5a451fcec49043f6b8476e151d1b/Model/MediaPlatform.php): Refactor this file to either declare symbols or cause side effects, but not both. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2036)
1. ![MAJOR][MAJOR] [MediaPlatform.php#L9](https://github.com/ExpandOnline/AuthManager/blob/b190584ccd8b5a451fcec49043f6b8476e151d1b/Model/MediaPlatform.php#L9): Refactor this class so it has no more than 20 fields, rather than the 21 it currently has. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS1820)
1. ![MINOR][MINOR] [GoogleDeveloperAuthManager.php](https://github.com/ExpandOnline/AuthManager/blob/b190584ccd8b5a451fcec49043f6b8476e151d1b/Lib/Google/GoogleDeveloperAuthManager.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)
1. ![MINOR][MINOR] [PagespeedAuthContainer.php](https://github.com/ExpandOnline/AuthManager/blob/b190584ccd8b5a451fcec49043f6b8476e151d1b/Lib/Pagespeed/PagespeedAuthContainer.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)
1. ![MINOR][MINOR] [PagespeedAuthManager.php](https://github.com/ExpandOnline/AuthManager/blob/b190584ccd8b5a451fcec49043f6b8476e151d1b/Lib/Pagespeed/PagespeedAuthManager.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)
1. ![MINOR][MINOR] [MediaPlatform.php](https://github.com/ExpandOnline/AuthManager/blob/b190584ccd8b5a451fcec49043f6b8476e151d1b/Model/MediaPlatform.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: Lib/Google/GoogleDeveloperAuthManager.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+App::uses('MediaPlatformAuthManager', 'AuthManager.Lib');
+App::uses('PagespeedAuthContainer', 'AuthManager.Lib/Facebook');
```
Comment (kulltc) [inline Lib/Google/GoogleDeveloperAuthManager.php]: Why?

File: Lib/Google/GoogleDeveloperAuthManager.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+abstract class GoogleDeveloperAuthManager extends MediaPlatformAuthManager {
```
Comment (kulltc) [inline Lib/Google/GoogleDeveloperAuthManager.php]: selfReferentialCallbackerinoManager

**Comment by pvaass** [comment]
@kulltc done

**Pull request merged**